plugins/diagnose.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: diagnose
+spec:
+  version: v0.1.1
+  homepage: https://github.com/alecjacobs5401/kubectl-diagnose
```

Comment:
off-topic: I recommend you list some example outputs so people can see how it works without installing.

Reply:
Thats a good suggestion, forgot to update my README with that!

Will do

Reply:
[Added!](https://github.com/alecjacobs5401/kubectl-diagnose/blob/master/README.md#example-usages)